### ISSUE TITLE
New TinyMCE version?

### ISSUE THREAD

**Issue by erral** [plone/mockup#840]
How can we update the version of TinyMCE? Some weeks ago we made an effort to complete the basque translation of TinyMCE and we would like to have it updated in Plone.

How can this be achieved?

**Comment by laulaz** [comment]
+1 here since there were many bugfixes since tinymce 4.7.6

Thank you !

**Comment by frapell** [comment]
@erral @laulaz Hey! upgrading tinymce is a bit tedious, here are the steps I have followed the last time:

1. Update https://github.com/artursmirnov/tinymce_builded with the latest version, following the steps explained there
2. Once it has been merged, update the version in mockup (Example https://github.com/plone/mockup/pull/832)
3. Make sure tests pass
4. Once mockup has the new version, update the CMFPlone files (Example https://github.com/plone/Products.CMFPlone/pull/2320)

Make sure you do it against master and 5.1.x branch, if you want to include this in both upcoming 5.2 and 5.1
